profanityfilter/profanityfilter.py

```diff
@@ -53,9 +53,30 @@ def append_words(self, word_list):
         """Define a custom list of profane words to be used in conjunction with the default list."""
         self._extra_censor_list.extend(word_list)
 
-    def remove_word(self, word):
-        """Remove given word from censor list."""
-        self._censor_list.remove(word)
+    def remove_word(self, word, anywhere=True):
+        """
+        Remove given word from censor list.
+            - anywhere (bool):
+                attempt removal from both the extra censor list and the custom/default censor list
+        """
+        if word in self._extra_censor_list and anywhere:
+            self._extra_censor_list.remove(word)
+        else:
+            if self._custom_censor_list:
```

Comment:
A test is failing: https://travis-ci.org/areebbeigh/profanityfilter/jobs/496166825#L488

```suggestion
            if word in self._custom_censor_list:
```

The final case is fine as is since we want an error to be thrown when the word isn't anywhere in the lists.